src/hevm/hevm-cli/hevm-cli.hs

```diff
@@ -749,14 +754,16 @@ vmFromCommand cmd = do
               then addr address (createAddress origin' (word nonce 0))
               else addr address 0xacab
 
-        vm0 miner ts blockNum diff c = EVM.makeVm $ EVM.VMOpts
+        vm0 baseFee miner ts blockNum diff c = EVM.makeVm $ EVM.VMOpts
```

Comment:
haskell question: why isn't the `baseFee` fetched from rpc above in scope here?

Reply:
because variables defined in `where` cannot "look inside" and see vars defined in the monadic context.